iroh/tests/cli.rs

```diff
@@ -478,6 +478,7 @@ fn run_cli(
 
 #[cfg(feature = "cli")]
 #[test]
+#[ignore = "flaky"]
```

Comment:
:( do we have an issue for this?

Reply:
#1935 

All these tests are fundamentally not reliable sadly.